aws/resource_aws_iam_service_linked_role_test.go

```diff
@@ -0,0 +1,90 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAwsIamServiceLinkedRole(t *testing.T) {
```

Comment:
Please rename this to `TestAccAWSIAMServiceLinkedRole_basic`. It should also have a test step for testing import (I'll note below)